services/batch.rst

```diff
@@ -0,0 +1,582 @@
+############################
+Using the LSST Batch Systems
+############################
+
+NCSA hosts a few `HTCondor <https://research.cs.wisc.edu/htcondor/>`_ clusters and a `Slurm <https://slurm.schedmd.com/overview.html>`_ cluster for LSST. Of particular interest to LSST DM developers will be:
+
+- ``HTCondor DAC Cluster`` - general user HTCondor pool
```

Comment:
I don't think there's any need to set these names in monospace. If you want to draw attention to them, probably better to use **bold**, or even a [definition list](https://developer.lsst.io/restructuredtext/style.html#definition-lists).

(Similar thing applies to repeated mention of them below).